source-code/sdk-js/src/adapter-sveltekit/ast-transformation/tests/hooks.server.js.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, it } from 'vitest'
+import {transformHooksServerJs, wrapHooksServerJs, createHooksServerJs } from '../transforms/hooks.server.js.js'
+import type {TransformConfig} from "../config.js";
+
+const baseConfig:TransformConfig = {
+    isStatic: false,
+    languageInUrl: false,
+    srcFolder: '',
+    rootRoutesFolder: '',
+    hasAlreadyBeenInitialized: false,
+}
+
+describe('transform hooks.server.js', () => {
```

Comment:
To give the file abit more structure please wrap all tests into a descrive with the name of the function you call in the test. e.g.

```ts
describe('transformHooksServerJs', () => {
    it.concurrent('transformHooksServerJs returns new file if no code provided', async ({ expect }) => {
        const code = transformHooksServerJs(baseConfig, '')
        expect(code).toContain('getLanguage')
    })
})
```

You can omit the outer `"transform hooks.server.js"`,